index.html

```diff
@@ -0,0 +1,328 @@
+                        <ul class="tag">
+                            <li> <a href="#">HTML</a></li>
+                            <li> <a href="#">Bootstrap</a></li>
+                            <li> <a href="#">Ruby</a></li>
+                        </ul>
+                    </div>
+
+                    <div class="btn2-holder">
+                        <a href="#" class="btn2">See Projects</a>
+                    </div>
+                </div>
+            </div>
+            <div class="grid-item item2">
+                <div class="content">
+                    <h1 class="p-text">Professional Art <br>
+                        Printing Data
+                    </h1>
+                    <h2 class="s-text">
+                        A daily selection of privately
+                        personalized reads; no accounts or
+                        sign-ups required. Has been the 
+                        industry's standard.
+                    </h2>
+                    <div class="tags">
+                        <ul class="tag">
+                            <li> <a href="#">HTML</a></li>
+                            <li> <a href="#">Bootstrap</a></li>
+                            <li> <a href="#">Ruby</a></li>
+                        </ul>
+                    </div>
+
+                    <div class="btn2-holder">
+                        <a href="#" class="btn2">See Projects</a>
+                    </div>
+                </div>
+            </div>
+            <div class="grid-item item2">
+                <div class="content">
+                    <h1 class="p-text">Professional Art <br>
+                        Printing Data
+                    </h1>
+                    <h2 class="s-text">
+                        A daily selection of privately
+                        personalized reads; no accounts or
+                        sign-ups required. Has been the 
+                        industry's standard.
+                    </h2>
+                    <div class="tags">
+                        <ul class="tag">
+                            <li> <a href="#">HTML</a></li>
+                            <li> <a href="#">Bootstrap</a></li>
+                            <li> <a href="#">Ruby</a></li>
+                        </ul>
+                    </div>
+
+                    <div class="btn2-holder">
+                        <a href="#" class="btn2">See Projects</a>
+                    </div>
+                </div>
+            </div>
+            <div class="grid-item item2">
+                <div class="content">
+                    <h1 class="p-text">Professional Art <br>
+                        Printing Data
+                    </h1>
+                    <h2 class="s-text">
+                        A daily selection of privately
+                        personalized reads; no accounts or
+                        sign-ups required. Has been the 
+                        industry's standard.
+                    </h2>
```

Comment:
> - [ ] I suggest you use a p tag instead of a heading tag. This way will be semantically correct and the screen readers and other technologies will thank you.😊